qemu/tests/hpt_miscellaneous.py

```diff
@@ -88,6 +89,8 @@ def run(test, params, env):
         # For HPT reszing after hotplug memory
         hpt_mem = MemoryHotplugTest(test, params, env)
         hpt_mem.hotplug_memory(vm, "hpt_mem")
+        wait_time = int(params.get("wait_time"))
```

Comment:
Since only  `with_mem_hotplug` has this params `wait_time`, other cases will return `wait_time = None`.
So better to:
1. Define a default value
```suggestion
        wait_time = int(params.get("wait_time", 0))
```
2. Make a judgment
```suggestion
        if params.get("wait_time")
            wait_time = int(params["wait_time"])
```